app/dashboard/views/fido_setup.py

```diff
@@ -91,6 +112,16 @@ def fido_setup():
     registration_dict = credential_create_options.registration_dict
     del registration_dict["extensions"]["webauthn.loc"]
 
+    # Prevent user from adding duplicated keys
+    for record in fido_model:
```

Comment:
`record` can be renamed to `fido` here :)

Reply:
https://github.com/simple-login/app/pull/199/commits/7bd97e13b008b4a3b265f44fc96c2cf8907b3b2a